src/test/java/io/github/tetherless_world/jena_elasticsearch/ElasticsearchGraphTest.java

```diff
@@ -1,21 +1,43 @@
 package io.github.tetherless_world.jena_elasticsearch;
 
 import junit.framework.TestSuite;
+import org.apache.http.HttpHost;
 import org.apache.jena.graph.Graph;
 import org.apache.jena.graph.test.AbstractTestGraph;
-import org.apache.jena.reasoner.test.TestInfGraph;
 
-public class ElasticsearchGraphTest extends AbstractTestGraph {
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.Random;
 
+public class ElasticsearchGraphTest extends AbstractTestGraph {
 
     public ElasticsearchGraphTest(String name) {
         super(name);
     }
 
-    public static TestSuite suite()
-    { return new TestSuite( ElasticsearchGraphTest.class ); }
+    public static TestSuite suite() {
+        return new TestSuite(ElasticsearchGraphTest.class);
+    }
 
     public Graph getGraph() {
```

Comment:
You seem to be missing @Override annotations. IntelliJ will add these on save with the "Save Actions" plugin.